src/algebra/group_with_zero/defs.lean

```diff
@@ -45,6 +45,14 @@ mul_zero_class.zero_mul a
 @[ematch, simp] lemma mul_zero (a : M₀) : a * 0 = 0 :=
 mul_zero_class.mul_zero a
 
+lemma pi.single_mul {ι : Type*} [decidable_eq ι] (i j : ι) (a : M₀) (f : ι → M₀) :
```

Comment:
```suggestion
lemma pi.single_mul {ι : Sort*} [decidable_eq ι] (i j : ι) (a : M₀) (f : ι → M₀) :
```
unless that doesn't work

Reply:
`pi.semiring` etc assume `ι` to be a `Type*`.